### PR TITLE
chore(cas): Warn if the tip isn't what we expect when loading stream at CommitID

### PR DESCRIPTION
**Pull request by stbrody**

### CODE REVIEW

File: src/services/anchor-service.ts [hunk 1]
```diff
@@ -326,6 +326,10 @@ export default class AnchorService {
         throw new Error(`No valid ceramic document found with docId ${docId.toString()}`)
       }
 
+      if (doc.tip.toString() != request.cid) {
```
Comment (oed) [inline src/services/anchor-service.ts:329]:
```suggestion
      if (doc.tip.toString() != request.cid.toString()) {
```
To be consistent with the logging below.

**Review by oed — COMMENTED** [review]
Question.

**Comment by stbrody** [comment]
> Question.

Question?

File: src/services/anchor-service.ts (hunk 1 above)
Comment (stbrody) [reply src/services/anchor-service.ts:329]: request.cid is already a string, not a CID object

File: src/services/anchor-service.ts [hunk 2]
```diff
@@ -326,6 +326,10 @@ export default class AnchorService {
...
+        logger.warn(`When loading document ${doc.id.toString()} at commit ${request.cid.toString()}, doc was returned at tip ${doc.tip.toString()}`)
```
Comment (oed) [inline src/services/anchor-service.ts:330]: Then why do we do toString here then?

**Review by oed — APPROVED** [review]

File: src/services/anchor-service.ts (hunk 2 above)
Comment (stbrody) [reply src/services/anchor-service.ts:330]: this is probably the part that should change I guess

Comment (stbrody) [reply src/services/anchor-service.ts]: Okay, updated to remove the toString here

**Pull request merged**